apps/frontend/src/app/5_pages/EarnPage/EarnPage.tsx

```diff
@@ -419,18 +418,11 @@ const EarnPage: FC = () => {
 
           <SimpleTable className="mt-3">
             <SimpleTableRow
-              label={
-                <span className="flex items-center gap-1">
-                  {t(pageTranslations.subsidiesRewardRate)}
-                  <HelperButton
-                    content={t(pageTranslations.subsidiesRewardRateInfo)}
-                  />
-                </span>
-              }
+              label={t(pageTranslations.subsidiesRewardRate)}
               valueClassName={classNames('transition-colors', {
                 'text-primary-10': !isAmountZero,
               })}
-              value={`${formatValue(apy, 2)}% APY`}
+              value={`${formatValue(apy, 2)}% APR`}
```

Comment:
```suggestion
              value={`${formatValue(apy, 2)}% APR`}
```

Please move the "APR" string to localisations file (or reuse existing one if appropriate)